crates/codegen/syntax_templates/src/rust/parser_output.rs

```diff
@@ -13,27 +10,34 @@ pub struct ParseOutput {
 
 impl ParseOutput {
     pub fn parse_tree(&self) -> Option<Rc<cst::Node>> {
-        self.parse_tree.clone()
+        return self.parse_tree.clone();
```

Comment:
nit: for short pure functions I prefer to leave the return off. Not an issue, just commenting.